### ISSUE TITLE
Create video demo of new features

### ISSUE THREAD

**Issue by reticulatedpines** [reticulatedpines/magiclantern_simplified#161]
A high quality video showing support has been extended to new cams, as well as any new features.  Possibly this splits nicely into two videos.

A lot of people have forgotten about ML, and it would be nice to remind people with a video shot on newly supported cams.

**Comment by WalterSchulz** [comment]
Repository for footage/edited content?
Creating high quality (= content quality in first place, right? Not only high-res/bit depth counts.) requires work. If a slight error occured - like using the wrong term for a button - it may not justify re-doing all the work from scratch but simply add a text overlay and re-upload content.
Doing so with YT content (example) will - if I got it right - mean working with compressed footage and re-uploading precompressed content does not sound like a good idea.

**Comment by reticulatedpines** [comment]
Yup, I don't care if it's shot in raw.  It's fun to shoot only with new cams, about new cams, and gives us an excuse to list new cams that have support.  Raw would be a nice bonus but I bet most people wouldn't even notice, since, as you say, they'll be watching it in some lossy output format.  I don't want to deal with the editing time!  Storyboarding and getting the framing + lighting nice is enough effort for me.

I guess I was thinking of a kind of "demo reel" format - a few seconds proving that each feature really exists.  5s shot of mov timer showing 60m, for example.  Then a pan around a 200d shooting in raw so you can see the cam model label, and the raw on the LCD.  Benchmark from dual card 7d2.  Etc.  This is "Make ML Cool Again".

A video tutorial is also valuable, I expect the format would be very different, thus likely a separate video.  Due to the greatly limited feature set for new cams, it's much easier to explore, so perhaps we need this less at this time.  And the old tutorials in theory cover it?
